source/add-existing.txt

```diff
@@ -31,31 +31,32 @@ Sinatra Application
 -------------------
 
 To start using {+odm+} in an existing Sinatra application, you can follow
-the same steps described in the
-:ref:`Sinatra Quick Start <mongoid-quick-start-sinatra>` guide.
-
-The following steps describe how to add {+odm+} to a Sinatra application:
+the following steps:
```

Comment:
S: change line to "perform the following steps:" or something similar to avoid double "follow"